terraform/production/main.tf

```diff
@@ -33,3 +33,14 @@ module "networking" {
   vpc_cidr_block                          = "10.0.0.0/16"
   vpc_flow_cloudwatch_log_expiration_days = 90
 }
+
+module "rds" {
+  source = "../modules/rds"
+
+  prefix               = "core-prod"
+  allocated_storage    = 100

```

Comment:
Looks like prod had ~100GiB of free space a year ago on PaaS (not sure if this was at inception or well used, as that's the furthest I could look), so I assumed this is the fresh disk space we'll want in AWS at minimum